City-of-Helsinki/helsinki-paatokset
# PP-400: Small fixes

**To test**
* Checkout branch, run `make drush-cr drush-cim`
* Go to https://helsinki-paatokset.docker.so/en/decision
  * The page should no longer have a duplicate main heading  
* Get some data for testing:
  * `drush ap:update organization U540200 -v`
  * `drush ap:update organization 00400 -v`
  * `drush ap:update organization 02900 -v`
  *  `drush ap:update meetings U540200202217`
  * `drush ap:update cases hel-2022-010676 -v`
  * `drush ap:update decisions a13de49c-893e-c65e-b410-838313500003 -v`
  * `drush ap:update decisions 59c384ab-528b-4403-acd2-f8401699d028 -v`
  * `drush ap:gm -v`
* Add a new minutes of dicussion document: https://helsinki-paatokset.docker.so/fi/media/add/minutes_of_the_discussion
  * Link it to "Kaupunkiympäristölautakunnan ympäristö- ja lupajaosto - 13.10.2022 - 16:15"
* Open the meeting page: https://helsinki-paatokset.docker.so/fi/paattajat/kaupunkiymparistolautakunnan-ymparisto-ja-lupajaosto/asiakirjat/U540200202217
  * There should be a "Kokousaika & Kokouspaikka" element below the title
  * The minutes of discussion PDF link should work (instead of being blank) 
* Open this decision: https://helsinki-paatokset.docker.so/fi/asia/hel-2022-010640?paatos=9165b5d0-c661-4c85-bdc3-4bd0bf87618f
  * The dropdown should be inactive because there's only one decision and it is currently selected
* Open this decision:  https://helsinki-paatokset.docker.so/fi/asia/hel-2022-010676?paatos=59c384ab-528b-4403-acd2-f8401699d028
  * The dropdown should work normally since there are multiple decisions for the case
